airbyte-integrations/connectors/source-sftp-bulk/source_sftp_bulk/spec.json

```
@@ -98,6 +98,14 @@
         "examples": ["2017-01-25T00:00:00Z"],
         "description": "The date from which you'd like to replicate data for all incremental streams, in the format YYYY-MM-DDT00:00:00Z. All data generated after this date will be replicated.",
         "order": 10
+      },
+      "separator": {
+        "title": "CSV Separator (Optional)",
+        "description": "The separator used in the cvs files. Define None if you want to use the Sniffer functionality",
+        "type": "string",
+        "default": ",",
+        "examples": [","],
+        "order": 1
```

`order` should be unique, it determines the order the config options will display. To set it as the last item displayed in the connector settings:

```suggestion
        "order": 11
```

Alternatively, you can adjust the `order` values in this file so it is further up.